examples/next-openai-telemetry/instrumentation.ts

```diff
@@ -0,0 +1,9 @@
+import { registerOTel } from '@vercel/otel';
+import { ConsoleSpanExporter } from '@opentelemetry/sdk-trace-node';
+
+export function register() {
+  registerOTel({
+    serviceName: 'next-app',
+    traceExporter: new ConsoleSpanExporter(), // TODO what exporter should I use here?
```

Comment:
For `@vercel/otel` you don't need an exporter - it configures automatically. Just `serviceName` is needed.

Reply:
How can I generate console output for checking in my local deployment without this?

Reply:
Hmm. That just doesn't really come up. Console output for tracing is just too spammy. I can potentially arrange something if it's very important. Ping me to discuss async.

Reply:
Maybe it's specific to my own use case (Otel development). It'll be rare that someone is working on OTel itself. I'll remove it.